src/DebuggerIdGenerator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Yiisoft\Yii\Debug;
+
+class DebuggerIdGenerator
```

Comment:
How about rename generator? It's stateful class that provides generated value. Usually generators provides `generate` method and are stateless.